openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.mode_changed.json

```diff
@@ -0,0 +1,34 @@
+{
+   "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+   "id":"urn:uuid:921bd866-93ac-4ed1-a147-cdd1fe91e87f",
+   "type":"Enrollment",
```

Comment:
Verify if this is a caliper supported `type`

Reply:
changed to ToolUseEvent